connect/api/src/main/java/org/apache/kafka/connect/data/SchemaProjector.java

```diff
@@ -37,13 +37,13 @@
 
 public class SchemaProjector {
 
-    private static Set<AbstractMap.SimpleImmutableEntry<Type, Type>> promotable = new HashSet<>();
+    private static final Set<AbstractMap.SimpleImmutableEntry<Type, Type>> PROMOTABLE = new HashSet<>();
```

Comment:
nit: We should probably call out that the casing of variable(s) has been changed  in the PR description? Currently it only says the usage of `final`..

Reply:
Also, wondering if this change got imposed due to a checkstyle?

Reply:
Added to the description and yes, checkstyle complained about it.